eng/native/build-commons.sh

```diff
@@ -104,14 +105,28 @@ build_native()
                 exit "$exit_code"
             fi
         else
-            # Generate the dummy version.c, but only if it didn't exist to make sure we don't trigger unnecessary rebuild
+            # Generate the dummy version.c and runtime_version.h, but only if they didn't exist to make sure we don't trigger unnecessary rebuild
             __versionSourceLine="static char sccsid[] __attribute__((used)) = \"@(#)No version information produced\";"
             if [[ -e "$__versionSourceFile" ]]; then
                 read existingVersionSourceLine < "$__versionSourceFile"
             fi
             if [[ "$__versionSourceLine" != "$existingVersionSourceLine" ]]; then
```

Comment:
Should be use the same update logic for both  version.c and runtime_version.h ?

Reply:
`version.c` is single line string. `runtime_version.h` is multiple macros, values of which much cast to `USHORT`. I couldn't figure out better way, but open for suggestions. 🙂

Reply:
I would change this to:

```
if [[ "$__versionSourceLine" != "$existingVersionSourceLine" ]]; then
   ... write runtimeVersionHeaderFile ...
   ... write versionSourceFile ...
fi
```

so that the two files are always in sync.

Reply:
Sure, I have place them both under the same condition.